template/index.ts

```diff
@@ -1,15 +1,20 @@
-function createSchemaObject(options) {
-  const { schemaName, type, isRequired, defaultValue } = options;
+export function createSchemaObject(options: {
```

Comment:
I feel like this typing was repeated, possibly here: https://github.com/eKhattak/mongoose-schema-cli/pull/13/files#diff-8cfead41d88ad47d44509a8ab0a109adR10

Instead of repeating, would be great to just use the same type you made at the link I mentioned. You can export and import types just like any other JS construct!